src/ec/packages/ec-component-accordion2/accordion2.html.twig

```diff
@@ -0,0 +1,92 @@
+{% spaceless %}
+
+{#
+  Parameters:
+    - "items" (array) (default: []): format: [
+      {
+        "id": (string),
+        level: (string),
+        toggle: (predefined structure): see Button component, format: {
+          label: (string),
+          icon: (predefined structure): see Icon component,
+        },
+        content: (block),
+      },
+      ...
+    ]
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-accordion2' %}
+{% set _extra_attributes = '' %}
+{% set _items = items|default([]) %}
+
+{# Internal logic - Process properties #}
+
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value ~ '"' %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+
+<div
+  class="{{ _css_class }}"{{ _extra_attributes|raw }}
+  data-ecl-accordion2="true"
+>
+  {% if _items is not empty %}
+    {% for _item in _items %}
+      <div class="ecl-accordion2__item">
+        <h{{ _item.level|default(3) }} class="ecl-accordion2__title">
+          <button
+            type="button"
+            class='ecl-accordion2__toggle'
+            data-ecl-accordion2-toggle
+            aria-controls="{{ _item.id }}-content"
+          >
+            <span class="ecl-accordion2__toggle-flex">
+              {% include '../ec-component-icon/icon.html.twig' with {
+                icon: {
+                  path: _item.toggle.icon.path,
+                  type: "ui",
+                  name: "plus",
+                  size: "m"
+                },
+                extra_classes: 'ecl-accordion2__toggle-icon',
+                extra_attributes: [{ name: "data-ecl-accordion2-icon", value: "true" }]
```

Comment:
it would be great if we didn't have to pass a `value` to extra attributes (unnecessary), but this change of logic would need to be applied to every components... unless we create some global macros

what do you think?

Reply:
actually, it does seems to work by putting only a name (without changing the logic). I'll update it